lib/rspec/core/formatters/message_formatter.rb

```diff
@@ -0,0 +1,28 @@
+module RSpec
+  module Core
+    module Formatters
+      # @api private
+      # Formatter for providing message output as a fallback when no other
+      # profiler implements #message
+      class MessageFormatter
```

Comment:
Since this isn't generally needed, and is only used as a fallback, WDYT of the name `FallbackMessageFormatter`?